matador/plotting/ir_plotting.py

```diff
@@ -0,0 +1,126 @@
+""" This submodule implements functions useful for plotting
+the results of infrared and Raman spectroscopy calculations.
+
+"""
+
+from matador.plotting.plotting import plotting_function
+
+
+def read_ir_file(seed):
```

Comment:
Looking at this, much of this functionality could be added on top of the `phonon2dict` scraper here: https://github.com/ml-evs/matador/blob/8b5316c4edc13173ad6069532f3c7c8ed5857c5a/matador/scrapers/castep_scrapers.py#L867

Do you want to merge it or shall I?

Reply:
I think it's probably best if you merge it. Chances are I'll make a mistake and you'll need to intervene anyway.